packages/math-input/src/components/input/math-input.js

```diff
@@ -239,15 +285,15 @@ class MathInput extends React.Component {
     };
 
     /** Gets and cache they bounds of the keypadElement */
-    _getKeypadBounds = () => {
+    _getKeypadBounds: () => $FlowFixMe = () => {
```

Comment:
I _think_ all these things dealing with bounds originate from a call to `getBoundingClientRect()` which is typed as returning a `DOMRect`.  Could we use that in this file?

Reply:
I couldn't get this to work and I don't want to linger too long on this. I was able to add `_keypadBounds: ?ClientRect;` but when I used `ClientRect` or `?ClientRect` as the return value here errors started to spread.